src/DurableTask.Netherite/Scaling/AzureTableLoadPublisher.cs

```diff
@@ -60,11 +65,11 @@ public Task PublishAsync(Dictionary<uint, PartitionLoadInfo> info, CancellationT
             }
             if (tableBatch.Count > 0)
             {
-                return this.table.SubmitTransactionAsync(tableBatch, cancellationToken);
-            }
-            else
-            {
-                return Task.CompletedTask;
+                var response = await this.table.SubmitTransactionAsync(tableBatch, cancellationToken);
+                foreach(var responseItem in response.Value)
+                {
+                    responseItem.Dispose();
+                }
```

Comment:
same feedback regarding `using`

Reply:
again, I see no technical way of using `using` as a means to call dispose here.